core/src/test/scala/unit/kafka/server/BaseRequestTest.scala

```diff
@@ -32,22 +32,19 @@ import org.apache.kafka.common.requests.{AbstractRequest, AbstractRequestRespons
 import org.apache.kafka.common.security.auth.SecurityProtocol
 
 abstract class BaseRequestTest extends IntegrationTestHarness {
-  override val serverCount: Int = numBrokers
   private var correlationId = 0
 
   // If required, set number of brokers
-  protected def numBrokers: Int = 3
+  override def serverCount: Int = 3
 
   // If required, override properties by mutating the passed Properties object
   protected def propertyOverrides(properties: Properties) {}
```

Comment:
I think this name doesn't quite work generally since `IntegrationTestHarness` exposes brokers and clients. So it should probably be renamed to make it clear that it's only for broker properties.

Reply:
That's a great catch, thanks